hil/ext/obm/ipmi.py

```diff
@@ -23,11 +23,21 @@
 from subprocess import call, Popen, PIPE
 import os
 
```

Comment:
note: since you modified this file here and in #835; you should rebase this PR just to be safe.